usermanagement/src/main/kotlin/com/story/usermanagement/storage/UserDynamoDto.kt

```diff
@@ -0,0 +1,32 @@
+package com.story.usermanagement.storage
+
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbSecondaryPartitionKey
+
+const val emailAddressIndex = "email-address-index"
+
+@DynamoDbBean
+@DynamoTable(baseTableName = "users")
+@Provisions(readCapacityUnits = 10L, writeCapacityUnits = 10L)
```

Comment:
What does this refers to?

Reply:
I created an annotation to set the read/write capacity.

The read/write capacity mode controls how you are charged for read and write throughput and how you manage capacity. 
I have specified the number of reads and writes per second  here for this table.